novels.sql

```diff
@@ -0,0 +1,149 @@
+
```

Comment:
This file can be deleted. We only used .sql files as a stand in until we learned migrations. In the real world you wouldn't use .sql files you'd use migrations from the beginning.